hvantk/tables/ucsc.py

```
@@ -7,24 +7,189 @@
+from scipy import sparse
 
 from hvantk.core.constants import UCSC_CELL_ID_COLUMN, UCSC_GENE_COLUMN
 
 __all__ = [
     "load_ucsc_metadata",
     "create_anndata_from_ucsc_matrix",
+    "build_ucsc_atlas_backed",
+    "summarize_ucsc_streaming",
+    "finalize_partial_atlas",
+    "coerce_obs_for_h5ad",
 ]
```

Ruff (0.15.10)

[warning] `__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)